skbio/alignment/_tabular_msa.py

```diff
@@ -1701,7 +1701,15 @@ def reassign_index(self, mapping=None, minter=None):
                 "Cannot use both `mapping` and `minter` at the same time.")
 
         if mapping is not None:
-            self._seqs.rename(mapping, inplace=True)
+            if isinstance(mapping, dict):
+                self.index = [mapping[label] if label in mapping else label
```

Comment:
I know this is merged already, but just took a peak. Just out of curiosity, is the ternary form favored over `mapping.get(label, label)` or does this fall to developer preference?